Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +121,24 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null || array.Length == 0)
+            {
+                return array;
+            }
+            if (index >= array.Length || index < 0)
+            {
+                return array;
+            }
+
+            var amendedArray = new int[array.Length - 1];
+            for (int i = 0, j = 0; i < array.Length; i++, j++)
+            {
+                if (i + 1 >= array.Length) break;
+                if (i == index) j++;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4904050)